packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -155,6 +155,14 @@ processors:
       field: email.subject
       copy_from: fortinet.firewall.subject
       if: "ctx?.fortinet?.firewall?.subject != null"
+  - set:
+      field: host.name
+      copy_from: fortinet.firewall.srcname
+      if: "ctx?.fortinet?.firewall?.srcname != null"
```

```suggestion
      if: "ctx.fortinet?.firewall?.srcname != null"
```
(similar throughout)